spaceship.zsh

```diff
@@ -179,6 +179,7 @@ spaceship_git_status() {
     indicators+="$(spaceship_git_untracked)"
     indicators+="$(spaceship_git_stashed)"
     indicators+="$(spaceship_git_unpushed_unpulled)"
+    indicators+="$(work_in_progress)"
```

Comment:
Probably, there's a need to move WIP logic in separate function, like `spaceship_git_wip`